docs/framework/windows-workflow-foundation/samples/built-in-activities.md

```diff
@@ -13,7 +13,7 @@ This section contains samples that demonstrate built-in Windows Workflow Foundat
  [Load From XAML](load-from-xaml.md)  
  Demonstrates how to dynamically load a XAML workflow without having to run the XamlBuildTask tool.
   
- [Using the InvokeMethod Activity](using-the-invokemethod-activity.md)  
+ [Using the InvokeMethod Activity](built-in-activities.md)  
```

Comment:
This just navigates back to the same page. We should remove this link and remove the page from the TOC.